src/main/scad/laszlo-molnar/l-i-vator-ii/molnar.l-i-vator-ii.scad

```diff
@@ -0,0 +1,82 @@
+include <puzzlecad.scad>
+
+box_puzzle_border = 6;
+
+$burr_inset = 0.125;
+$burr_bevel = 1;
+$unit_beveled = true;
+$burr_scale = 17;
+
+dim = $burr_scale * 3 + box_puzzle_border * 2 + $burr_inset * 2;
+height = $burr_scale * 3 + box_puzzle_border + $burr_inset * 2;
+
+mid = box_puzzle_border / 2 + 0.5;
+far = dim - mid;
+    
+*box();
+*pieces();
+*cap();
+
+module box() {
+    render(convexity = 2)
+    difference() {
+        beveled_cube([dim, dim, height]);
+        translate([box_puzzle_border, box_puzzle_border, box_puzzle_border])
+        cube([
+            $burr_scale * 3 + $burr_inset * 2,
+            $burr_scale * 3 + $burr_inset * 2,
+            $burr_scale * 4
+        ]);
+     }
+     
+     translate([0, 0, height])
+     connector(1.8, 2.0);
+}
+
+module pieces() {
+    burr_plate([
+        ["xx|xx", ".x|x.", "..|x."],
+        ["x.|xx", "..|x.", "..|x."],
+        ["xx|xx", "..|x.", "..|x."],
```

Comment:
These pieces are currently modeled with height 3... would it make sense to re-orient them to lay them flatter (for ease of printing / fewer layers)?

Reply:
Because there is 100*100*100 platform in my printer, I would like to use less space for each pieces in my model. But your comment is also very reasonable, I will update later.

Reply:
Because of small platform, I need to separate and rearrange all pieces in the clip tool eventually, so I can change the orientation of some pieces in Openscad.

Reply:
because of below piece, it will need support or joint if I change the orientation.
![image](https://user-images.githubusercontent.com/1962079/75249269-ef8df380-5810-11ea-92d0-0a21fdd90ce0.png)

Because height 3 is needed for this piece, I think we don't need to change the orientation of other height 3 pieces.

Reply:
Ah you're right. This piece needs height 3 (and printing at height 3 is preferred to splitting it up)